scss/tech.scss

```diff
@@ -1,50 +1,48 @@
 $tech-logo-size: 100px;
-$tech-span-width: 200px;
-$tech-span-height: 300px;
-$max-percentaje: 100%;
-$border-radius: 10px;
 
 .tech-div {
-  margin-left: 100px;
   display: flex;
-  justify-content: flex-start;
-  flex-direction: row;
+  flex-wrap: wrap;
+  margin-left: 100px;
 }
 
-.tech-span {
+.tech-element-div {
+  align-items: center;
   background-color: $green;
-  border-radius: $border-radius;
-  width: $max-percentaje;
-  max-width: $tech-span-width;
-  height: $tech-span-height;
-  max-height: $max-percentaje;
-  position: relative;
-  margin: 15px;
+  border-radius: 10px;
   display: flex;
-  align-items: center;
-  justify-content: flex-start;
   flex-direction: column;
+  height: 350px;
+  justify-content: space-evenly;
+  max-width: 225px;
+  max-height: 100%;
+  margin: 15px;
+  padding-bottom: 15px;
+  padding-left: 15px;
+  padding-right: 15px;
+  position: relative;
+  width: 100%;
 
   &::before {
-    content: '';
     border: 2px solid $green;
-    border-radius: $border-radius;
+    border-radius: 10px;
+    bottom: 5px;
+    content: '';
+    height: 350px;
+    max-width: 225px;
+    max-height: 100%;
```

Comment:
I rather setting max-height as fixed (in px) and height as 100%. Its easier to understand